MSProject/ViewControllers/DetailBook/View/WBDetailBookHeaderView.swift

```diff
@@ -30,8 +30,6 @@ class WBDetailBookHeaderView: UIView, NibLoadable {
     }
     
     func configureUI() {
-        rentButton.setTitle("RENT_BOOK_BUTTON".localized(), for: .normal)
-        wishlistButton.setTitle("WISHLIST_ADD_BUTTON".localized(), for: .normal)
         customBackgroundView.layer.cornerRadius = 5
         customBackgroundView.backgroundColor = .white
```

Comment:
for static configurations like this we like to put them on the didSet of the afected view to keep it cleaner, like this:

```
@IBOutlet private weak var customBackgroundView: UIView! {
        didSet {
                customBackgroundView.layer.cornerRadius = 5        
                customBackgroundView.backgroundColor = .white
              }
    }
       
```